### PR TITLE
BaSP-TG-126: Unify Code Redux

### PR DESCRIPTION
**Pull request by matescarabino**
- Unify redux constants names
- Unify redux actions
- Unify redux  reducers
- Unify redux thunks
- Fix all broken files
- Add new "shared" file to redux, modals are now stored there.

QA:
How to test?
I will add this after consulting with my teammates

### CODE REVIEW

File: src/redux/employees/reducer.js [hunk 1]
```diff
@@ -98,16 +76,22 @@ const employeesReducer = (state = INITIAL_STATE, action) => {
         ...state,
         isLoading: false,
         showConfirmModal: false,
-        modalContent: { title: 'SUCCESS!', content: `Employee Successfully CREATED` },
-        showModalMessage: true,
-        list: [...state.list, action.payload]
+        list: [...state.list, action.payload],
+        modalContent: {
+          title: 'SUCCESS!',
+          content: `Employee Successfully CREATED`
```
Comment (Ansitox) [inline src/redux/employees/reducer.js:82]: Change the backticks

File: src/redux/employees/reducer.js [hunk 2]
```diff
@@ -120,41 +104,48 @@ const employeesReducer = (state = INITIAL_STATE, action) => {
         ...state,
         isLoading: false,
         showConfirmModal: false,
-        modalContent: { title: 'SUCCESS!', content: `Employee Successfully UPDATED` },
+        modalContent: {
+          title: 'SUCCESS!',
+          content: `Employee Successfully UPDATED`
```
Comment (Ansitox) [inline src/redux/employees/reducer.js:109]: Change the backticks

File: src/redux/projects/reducer.js [hunk 3]
```diff
@@ -39,147 +35,118 @@ const projectsReducer = (state = INITIAL_STATE, action) => {
...
+    case PUT_PROJECTS_SUCCESS:
       return {
         ...state,
-        list: [...state.list, action.payload],
         isLoading: false,
         showConfirmModal: false,
         modalContent: {
           title: 'SUCCESS!',
-          content: action.msg
+          content: `Project Successfully UPDATED`
```
Comment (Ansitox) [inline src/redux/projects/reducer.js:109]: Change the backticks

File: src/redux/super-admins/reducer.js [hunk 4]
```diff
@@ -31,78 +27,79 @@ const INITIAL_STATE = {
...
-    case POST_SUPER_ADMINS_ERROR:
+    case POST_SUPERADMINS_ERROR:
       return {
+        ...state,
         isLoading: false,
-        error: action.payload,
         showConfirmModal: false,
         modalContent: {
           title: 'ERROR!',
           content: `Could not CREATE SuperAdmin`
```
Comment (Ansitox) [inline src/redux/super-admins/reducer.js:93]: Change the backticks

File: src/redux/super-admins/reducer.js [hunk 5]
```diff
@@ -114,72 +111,42 @@ const superAdminsReducer = (state = INITIAL_STATE, action) => {
...
       };
-    case DELETE_SUPER_ADMINS_ERROR:
+    case DELETE_SUPERADMINS_ERROR:
       return {
         ...state,
         isLoading: false,
-        error: action.payload,
         modalContent: {
           title: 'ERROR!',
           content: `Could not DELETE SuperAdmin`
```
Comment (Ansitox) [inline src/redux/super-admins/reducer.js:146]: Change the backticks

File: src/redux/tasks/reducer.js [hunk 6]
```diff
@@ -46,121 +42,111 @@ const tasksReducer = (state = INITIAL_STATE, action) => {
...
       return {
         ...state,
         isLoading: false,
-        list: [...state.list.filter((task) => task._id !== action.payload)],
+        showModalConfirm: false,
         modalContent: {
-          title: 'SUCCESS',
-          content: `Task with id ${action.payload} successfully deleted`
+          title: 'SUCCESS!',
+          content: `Task Successfully CREATED`
```
Comment (Ansitox) [inline src/redux/tasks/reducer.js:81]: Change the backticks

File: src/redux/tasks/reducer.js [hunk 7]
```diff
@@ -46,121 +42,111 @@ const tasksReducer = (state = INITIAL_STATE, action) => {
...
-    case CREATE_TASKS_SUCCESS:
+    case PUT_TASKS_SUCCESS:
       return {
         ...state,
         isLoading: false,
         showModalConfirm: false,
-        modalContent: { title: 'SUCCESS!', content: `Task Successfully CREATED` },
+        modalContent: {
+          title: 'SUCCESS!',
+          content: `Task Successfully UPDATED`
```
Comment (Ansitox) [inline src/redux/tasks/reducer.js:109]: Change the backticks

File: src/redux/timesheets/reducer.js [hunk 8]
```diff
@@ -109,19 +79,18 @@ const timesheetsReducer = (state = INITIAL_STATE, action) => {
         showConfirmModal: false,
         modalContent: {
           title: 'SUCCESS!',
-          content: action.reqMessage
+          content: `TimeSheet Successfully CREATED`
```
Comment (Ansitox) [inline src/redux/timesheets/reducer.js:82]: Change the backticks

File: src/redux/timesheets/reducer.js [hunk 9]
```diff
@@ -138,49 +107,45 @@ const timesheetsReducer = (state = INITIAL_STATE, action) => {
         showConfirmModal: false,
         modalContent: {
           title: 'SUCCESS!',
-          content: action.reqMessage
+          content: `TimeSheet Successfully UPDATED`
```
Comment (Ansitox) [inline src/redux/timesheets/reducer.js:110]: Change the backticks

**Review by Ansitox — APPROVED** [review]
Looks great to me, nicely done!

**Review by CristianBucci — DISMISSED** [review]
Everything looks right!

**Review by CristianBucci — CHANGES_REQUESTED** [review]
Buttons don't work

**Pull request closed without merging**